src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import usersFromServer from '../../api/users';
+import React from 'react';
+import { TodoType } from '../../types/type';
+
+type Props = {
+  todo: TodoType;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const users = usersFromServer.find(user => user.id === todo.userId);
```

Comment:
```suggestion
  const user = usersFromServer.find(user => user.id === todo.userId);
```
`find` method returns only one element